py/dml/types.py

```diff
@@ -610,6 +612,29 @@ def clone(self):
     def declaration(self, var):
         return ('ssize_t ' if self.signed else 'size_t ') + var
 
+class TInt64_t(IntegerType):
+    '''The '[u]int64_t' type from ISO C. For compatibility with C
+    APIs, e.g., calling an externally defined C function that takes a
+    `uint64_t *` arg. We find `uint64` a generally more useful type
+    primarily because it works better with format strings; "%lld" works
+    for `uint64` on both linux64 and win64.
+    '''
+    __slots__ = ()
+    def __init__(self, signed, const=False):
+        IntegerType.__init__(self, 64, signed, const=const)
+
+    def describe(self):
+        return self.declaration('').rstrip()
```

Comment:
ew. I'd vote for repetition instead (`return 'int64_t' if self.signed else 'uint64_t`), or, if you want to be particularly snazzy, leverage my "favorite" trick: `return 'u'*(not self.signed) + 'int64_t'`.

Reply:
factored out a function